projects/packages/publicize/src/class-rest-controller.php

```diff
@@ -82,6 +92,27 @@ public function get_publicize_connections() {
 		return rest_ensure_response( $this->make_proper_response( $response ) );
 	}
 
+	/**
+	 * Gets the publicize shares count for the site.
+	 *
+	 * GET `jetpack/v4/publicize/shares-count`
+	 */
+	public function get_publicize_shares_count() {
+		$blog_id  = $this->get_blog_id();
+		$response = Client::wpcom_json_api_request_as_blog(
```

Comment:
I've just been testing this and the results from this call end up being 0 but if I call the API endpoint myself, I get the right count coming back.

I think it might be that the ES library restricts the results to blogs that the current user has access to, and there is no current user with this call.

Perhaps we shauld go back to `wpcom_json_api_request_as_user`? Don't hate me!

Either that or maybe we can get the blog ID in that list when the current user ID is 0?

Reply:
No problem at all, Paul! 

Thanks for figuring out the solution here for recommending to use the internal_log strategy. Pasting the slack thread here for context. p1655734931557879/1655709923.750489-slack-C02JJ910CNL

I reckon we can mark this conversation as resolved?